src/RFC-0201_TariScript.md

```diff
@@ -279,7 +415,7 @@ Note that:
 
 ### Transaction input changes
 
-The current definition of an input is
+The definition of a Tari _script-less_ input would be:
```

Comment:
As above. Things are stabilised now, so just change language to reflect the current state of things